frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -25,8 +25,15 @@ export const surveysLogic = kea<surveysLogicType>([
         surveys: {
             __default: [] as Survey[],
             loadSurveys: async () => {
-                const response = await api.surveys.list()
-                return response.results
+                const responseSurveys = await api.surveys.list()
+                const surveysResponsesCount = await api.surveys.getResponsesCount()
```

Comment:
I'd recommend making this a separate loader and async so we don't block loading the list for the responses count.

The list is super fast since it's a postgres direct access, while the response count can be much slower depending on time range / no. of events / clickhouse load etc.